dev/com.ibm.ws.messaging.open_jms20context_fat/fat/src/com/ibm/ws/messaging/JMS20/fat/JMSContextTest_118061.java

```diff
@@ -0,0 +1,467 @@
+    }
+
+    // 118061_12 Verify set and get operation on Message header field
+    // JMSTimeStamp
+
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSTimestamp_B_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSTimestamp_B_SecOff");
+        assertTrue("Test testJMSTimestamp_B_SecOff failed", testResult);
+    }
+
+    // TCP and Security Off
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSTimestamp_TCP_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSTimestamp_TCP_SecOff");
+        assertTrue("Test testJMSTimestamp_TCP_SecOff failed", testResult);
+    }
+
+    // 118061_13 Verify set and get operation on Message header field
+    // JMSCorrelationID
+
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSCorrelationID_B_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSCorrelationID_B_SecOff");
+        assertTrue("Test testJMSCorrelationID_B_SecOff failed", testResult);
+    }
+
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSCorrelationID_TCP_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSCorrelationID_TCP_SecOff");
+        assertTrue("Test testJMSCorrelationID_TCP_SecOff failed", testResult);
+    }
+
+    // 118061_14 Verify set and get operation on Message header field
+    // JMSCorrelationIDAsBytes
+
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSCorrelationIDAsBytes_B_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSCorrelationIDAsBytes_B_SecOff");
+        assertTrue("Test testJMSCorrelationIDAsBytes_B_SecOff failed", testResult);
+    }
+
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSCorrelationIDAsBytes_TCP_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSCorrelationIDAsBytes_TCP_SecOff");
+        assertTrue("Test testJMSCorrelationIDAsBytes_TCP_SecOff failed", testResult);
+    }
+
+    // 118061_15 Verify set and get operation on Message header field JMSReplyTo
+
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSReplyTo_B_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSReplyTo_B_SecOff");
+        assertTrue("Test testJMSReplyTo_B_SecOff failed", testResult);
+    }
+
+    @Mode(TestMode.FULL)
+    @Test
+    public void testJMSReplyTo_TCP_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSReplyTo_TCP_SecOff");
+        assertTrue("Test testJMSReplyTo_TCP_SecOff failed", testResult);
+    }
+
+    // 118061_16 Verify set and get operation on Message header field
+    // JMSRedelivered
+
+    // @Test
+    public void testJMSRedelivered_B_SecOff() throws Exception {
+        boolean testResult = runInServlet("testJMSRedelivered_B_SecOff");
+        assertTrue("Test testJMSRedelivered_B_SecOff failed", testResult);
+    }
+
+    //@Test
```

Comment:
Fyi, If you scan for pattern `// @Test` when enabling specific test methods, you might miss this one.

Reply:
Noted.  I'm correcting that comment to '// @Test' to be consistent with the other disabled test.
I checked the original tests, and the two disabled tests were originally disabled.  I'm adding a comment to note that.